os/drivers/testcase/kernel_testcase_drv.c

```diff
@@ -153,5 +204,5 @@ static ssize_t kernel_test_drv_write(FAR struct file *filep, FAR const char *buf
 
 void kernel_test_drv_register(void)
 {
-	(void)register_driver(KERNEL_TC_DRVPATH, &kernel_test_drv_fops, 0666, NULL);
+	(void)register_driver(TESTCASE_DRVPATH, &kernel_test_drv_fops, 0666, NULL);
```

Comment:
Please change the function name and operation variable name also.

Reply:
Done.